gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Wed Dec 14 12:53:56 CET 2016
+#Fri Mar 03 09:38:24 GMT 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.2.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-all.zip
```

Comment:
Yes, 3.4 provides significant improvements: https://docs.gradle.org/current/release-notes.html But there seems to be an issue with the publishing plugin. Not sure about that though.